src/wiki/ventoy.md

```diff
@@ -0,0 +1,42 @@
+# Creating a Multi-OS Bootable Drive (with Ventoy)
+In this article, you will learn how to create a multi-OS bootable USB drive. using Ventoy 
+
+Ventoy is a tool that allows you to create a bootable USB drive, with multiple operating systems or recovery tools. Unlike other software, Ventoy allows you to simply store the ISO files **directly** on the USB. This can be really helpful if you have a suite of tools that you often use and you don't want to use a different USB drive for each one of them.
+
+## Creation of the USB
+In this section, you will learn how to use the Ventoy-to-USB tool, to install Ventoy on your USB.
+
+::: danger
+Installing Ventoy to create a bootable drive **will wipe all your data from that drive**. Back up your data on the USB drive before proceeding.
+:::
+1. Download the Ventoy-to-USB tool here [here](https://www.ventoy.net/en/download.html). Since Ventoy is open-source, [you can also download the latest release from GitHub](https://github.com/ventoy/Ventoy/releases).
```

Comment:
It might be better to be more descriptive with what you're hyperlinking here. You also said `here` twice. Is it necessary to mention that it is open source? How does this help readers who are following the steps?
```suggestion
1. Download the [Ventoy-to-USB tool](https://www.ventoy.net/en/download.html). Since Ventoy is open-source, [you can also download the latest release from GitHub](https://github.com/ventoy/Ventoy/releases).
```